tests/test_webgl_context_attributes_common.c

```diff
@@ -266,17 +266,19 @@ static int resultStencil = 0;
 static int resultAlpha = 0;
 
 static void draw() {
-  
   if (!resultAA) resultAA = testAntiAliasing(antiAliasingActivated);
+  assert(resultAA == 1);
```

Comment:
Perhaps the `== 1` can be removed, as I see they return ints, and the final result later down just checks they are not 0.

The test functions should probably return `bool` instead of int, which would remove the ambiguity.

Reply:
Good idea. I'm not sure how REPORT_RESULT works with bools, but I assume I don't need to change anything else.